pycds/climate_baseline_helpers.py

```diff
@@ -177,3 +177,130 @@ def parse_line(line):
     print('{} input lines skipped'.format(n_skipped))
 
     return (n_added, n_skipped)
+
+
+def expect_value(what, value, expected):
+    assert value == expected, \
+        '{}: expected "{}", got "{}"'.format(what, expected, value)
+
+
+def verify_baseline_network_and_variables(session):
+    """Verify that the expected baseline network and variable records are present in database.
+    These reproduce most of the unit tests for the corresponding upload functions.
+
+    :param session:
+    :return: boolean (always True)
+    :raise AssertionError when any error is detected
+    """
+
+    def expect_variable(name):
+        variable = session.query(Variable) \
+            .join(Variable.network) \
+            .filter((Network.name == pcic_climate_variable_network_name) & (Variable.name == name)) \
+            .first()
+        assert variable, 'Climate baseline variable named "{}" not found in database'.format(name)
+        return variable
+
+
+    def expect_variable_attr(variable, attr, expected):
+        value = getattr(variable, attr)
+        expect_value('Variable "{}" {}'.format(variable.name, attr), value, expected)
+
+    # Network
+    networks = session.query(Network).filter(Network.name == pcic_climate_variable_network_name)
+    assert networks.count() == 1, 'Network "{}" not found in database'.format(pcic_climate_variable_network_name)
+    network = networks.first()
+
+    # Temperature variables
+    for (name, keyword, kwd) in [
+        ('Tx_Climatology', 'maximum', 'Max.'),
+        ('Tn_Climatology', 'minimum', 'Min.'),
+    ]:
+        temp_variable = expect_variable(name)
+        expect_variable_attr(temp_variable, 'unit', u'celsius')
```

Comment:
You shouldn't need to use ``u''`` style unicode strings. Those were a vestige of the first couple minor versions of Python 3 (3.0, 3.1) that everyone realized was a mistake and never used again. E.g.

```python
$ python2.7
Python 2.7.12 (default, Nov 29 2016, 16:30:36) 
[GCC 4.9.3] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> u'foo' == 'foo'
True
$ python3.3
Python 3.3.5 (default, May 10 2016, 10:39:16) 
[GCC 4.9.3] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> u'foo' == 'foo'
True
```